org.scala-ide.sdt.core.tests/src/org/scalaide/core/classpath/ClasspathTests.scala

```diff
@@ -522,8 +522,9 @@ class ClasspathTests {
     checkMarkers(0, 0)
 
     // two excepted code errors
-    var markers= project.underlying.findMarkers(SdtConstants.ProblemMarkerId, true, IResource.DEPTH_INFINITE)
-    assertEquals("Unexpected number of scala problems in project", 2, markers.length)
+    var markers = project.underlying.findMarkers(SdtConstants.ProblemMarkerId, true, IResource.DEPTH_INFINITE)
+    val errors = SDTTestUtils.userFriendlyMarkers(markers.toList)
+    assertEquals("Unexpected number of scala problems in project: " + errors, 2, errors.length)
```

Comment:
I agree with you in principle, but as I already mentioned, the issue is that some error messages look different in 2.11 and 2.10. Text matching will not always work. Currently we only build with 2.11, so maybe when we move to 2.12 we'll have some work to do.

If it's not too much work, we can test the errors themselves, but I'm also ok with just displaying them.